pkg/kubectl/resource/builder.go

```diff
@@ -184,6 +184,15 @@ func (b *Builder) ResourceTypes(types ...string) *Builder {
 	return b
 }
 
+// ResourceNames accepts a default type and one or more names, and creates tuples of
+// resources
+func (b *Builder) ResourceNames(resource string, names ...string) *Builder {
+	for _, name := range names {
```

Comment:
Exactly

On Jul 12, 2015, at 1:46 PM, Michail Kargakis <notifications@github.com>
wrote:

In pkg/kubectl/resource/builder.go
<https://github.com/GoogleCloudPlatform/kubernetes/pull/10979#discussion_r34423735>
:

> @@ -184,6 +184,15 @@ func (b *Builder) ResourceTypes(types ...string) *Builder {
>  	return b
>  }
>
> +// ResourceNames accepts a default type and one or more names, and creates tuples of
> +// resources
> +func (b *Builder) ResourceNames(resource string, names ...string) *Builder {
> +	for _, name := range names {

Ok, I see that this has to be the equivalent of ResourceTypeOrNameArgs for
names

—
Reply to this email directly or view it on GitHub
<https://github.com/GoogleCloudPlatform/kubernetes/pull/10979/files#r34423735>
.